nixos/modules/services/x11/desktop-managers/default.nix

```diff
@@ -26,6 +26,31 @@ in
 
     services.xserver.desktopManager = {
 
+      wallpaper = {
+        mode = mkOption {
+          type = types.enum [ "center" "fill" "max" "scale" "tile" ];
+          default = "scale";
+          example = "fill";
+          description = ''
+            Wallpaper image mode:
```

Comment:
Yes, I agree.
It took me a while to figure out how to set the background image.

The background image path should probably be an option.
This would add the possibility of multiple images, one for each monitor. (with `combineScreens` disabled)